content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md

```diff
@@ -0,0 +1,101 @@
+
+Examples:
+
+* Smart Array controller firmware
+* Network Interface Cards (NICs) firmware
+
+> Note: devices depending associated to a type D firmware update are called *type-D devices*.
+
+## Added value of PLDM for FWUPD
+
+When PLDM for FWUPD is not implemented in a Redfish service (i.e. iLO 5 firmware prior to version 2.30), or, if a type D firmware package does not support PDLM for FWUPD, the target device is flashed by a run time agent (SUM or SUT). If no run time agent is present, the firmware cannot be updated as explained in the second  [part](https://developer.hpe.com/blog/hpe-firmware-updates-part-2-interaction-in-operating-modes/) of the above blog post list.
+
+When PLDM for FWUPD is supported by both the iLO Redfish service and a firmware package of a type-D device, the firmware package falls in type A. As a result the iLO can flash the bits and no server reboot is required to activate them anymore.
+
+The advantage of the move from type D to type A has several important consequences:
+
+* No downtime of the server
+* Faster update process
+* No need to have an operating system installed
+* No need of run time agents
+
+### How do I recognize a PLDM for FWUPD capable package ?
+
+PLDM for FWUPD capable packages have a `.fwpkg` extension. Some of them specifically mention `PLDM` or `pldm` in their filename like the HPE MCX512F-ACHT Mellanox Adapter package:  `16_32_1010-MCX512F-ACH_Ax_Bx.pldm.fwpkg`. However, to be sure that a firmware package supports PLDM for FWUPD, you have to extract its `payload.json` file and look for the `UpdatableBy` and `PLDMImage` key/values as shown in the next picture. The content of this file clearly mentions that the firmware is a PLDM image and that it can be updated by the Baseboard Management Controller (BMC). It also specifies that no reset of the server is required, which confirms that this context is of type A.
+
+![Partial extract of a PLDM for FWUPD payload.json file](/img/payloadofpldmimage.png "Partial extract of a PLDM for FWUPD payload.json file")
+
+## Conclusion
+
+By  transforming type-D devices into type-A, the Platform Level Data Model for Firmware Update specification provides a real benefit to end customers as no more server reboot is required. Firmware providers, too, take advantage of this new standard, as they don't need to adapt their flash tools to each computer integrators supporting their devices. Finally, computer makers don't need to provide run time agents supporting firmware provider flash tools.
+
+In addition to the drastic simplification and enhancement of firmware management, the PLDM DMTF standard enhances and simplifies external device supplied devices, in terms of monitoring and configuration. Read the [PLDM for RDE](TBD) article to better understand how.
+
+As previously mentioned, 
+Don't forget to check out some of my other [blog posts](https://developer.hpe.com/search/?term=donze) on the HPE Developer portal to learn more about Redfish tips and tricks.
```

Comment:
In the preview, these two lines show as one sentence, so you have:
As previously mentioned, Don't forget

I think you meant to do that, but you should change "Don't" to "don't"

Reply:
Done.